recipes/primer3-py/meta.yaml

```diff
@@ -10,7 +10,7 @@ source:
   sha256: 1ec6ce99ea149b1c82361c654a0eab824e698ce32d5f9ed2dd8f8f37564b3f5f
 
 build:
-  number: 2
+  number: 3
   skip: True  # [py < 38]
```

Comment:
```suggestion
```
This actually can be removed since we don't build Python < 3.8 any more.

Reply:
@aliciaaevans 
Thank you so much for your reviewing!